### DISCUSSION TITLE
input to pathfindR

### DISCUSSION THREAD

**Discussion by masaver** [egeulgen/pathfindR#60] [Q&A] [answered]
Hi Ege,

More than an issue , I have a question: After reading your paper and the documentation of pathfindR, I understand that you  use as an input any OMICs data  for which you can run a "differential expression/abundance" analysis . Is this correct ?

Also, when using your OMICs data with the score_terms() function: is there any data normalization method that you recommend?

Cheers,

-Mathias

**Comment by egeulgen** [comment, marked as answer]
Hey Mathias,

You understood correctly. More generally, any data containing genes and p values associated with these genes (two-class comparison, WGCNA gene importance significances etc.) can be used with pathfindR.

For `score_terms()`, we do not have any specific recommendation for data normalization. As long as distributions by samples are roughly equivalent, the method should provide reliable results. You can read how aggregated term scoring is performed [here](https://github.com/egeulgen/pathfindR/wiki/Enriched-Term-Scoring)

Hope this helps,
Best,
-E
